docs/book/src/topics/managedcluster.md

```diff
@@ -312,6 +312,34 @@ spec:
     maxSize: 10
 ```
 
+The cluster autoscaler behavior settings can be set in the `AzureManagedControlPlane`. Not setting a property will default to the value used by AKS. All values are expected to be strings.
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureManagedControlPlane
+metadata:
+  name: my-cluster-control-plane
+spec:
+  autoscalerProfile:
+    balanceSimilarNodeGroups: "false"
+    expander: "random"
+    maxEmptyBulkDelete: "10"
+    maxGracefulTerminationSec: "600"
+    maxNodeProvisionTime: "15m"
+    maxTotalUnreadyPercentage: "45"
+    newPodScaleUpDelay: "0s"
+    okTotalUnreadyCount: "3"
+    scanInterval: "10s"
+    scaleDownDelayAfterAdd: "10m"
+    scaleDownDelayAfterDelete: "10s"
+    scaleDownDelayAfterFailure: "3m"
+    scaleDownUnneededTime: "10m"
+    scaleDownUnreadyTime: "20m"
+    scaleDownUtilizationThreshold: "0.5"
+    skipNodesWithLocalStorage: "true"
```

Comment:
this doc disagrees with the default applied below

Reply:
fixed this doc so it matches what is actually applied by AKS rather than the defaults from the cluster-autoscaler docs